packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -130,6 +131,10 @@ const Signin = ({
           </div>
         </form>
 
+        {/* TODO: We will need to pull the enabled flag from feature flags or experiment data
+         */}
+        <ThirdPartyAuth {...{ enabled: false }}></ThirdPartyAuth>
```

Comment:
You could also self-close this tag, like we did above on https://github.com/mozilla/fxa/blob/11504ca0767a4845728c53acc9b6ea39340a4204/packages/fxa-settings/src/components/ThirdPartyAuth/mocks.tsx#L9

```suggestion
        <ThirdPartyAuth {...{ enabled: false }} />
```